lib/utils/common.py

```diff
@@ -2340,31 +2339,40 @@ def _zip_files(dir_path, _size=1):
                     os.remove(_file)
                 except Exception as e:
                     print(e)
-                    pass
 
 
-def get_system_commands(port=3000):
+def get_system_commands(port=3000) -> list[list[str]]:
+    """
+    Returns list of system commands and their alternatives to gather output from. The
+    unparsed output is then stored in a sysinfo.log. This list of commands is separate
+    from what is found in class Node which are system commands which are parsed and stored
+    as json to later be used in collectinfo mode's summary and health commands.
+    """
+
     # Unfortunately timestamp cannot be printed in Centos with dmesg,
     # storing dmesg logs without timestamp for this particular OS.
     if "centos" == (distro.linux_distribution()[0]).lower():
-        cmd_dmesg = "sudo dmesg"
-        alt_dmesg = ""
+        dmesg_cmds = ["sudo dmesg"]
+    else:
+        dmesg_cmds = ["sudo dmesg -T", "sudo dmesg"]
+
+    if "darwin" in platform.system().lower():
+        netstat_cmds = ["netstat -rn"]
     else:
-        cmd_dmesg = "sudo dmesg -T"
-        alt_dmesg = "sudo dmesg"
+        netstat_cmds = ["netstat"]
 
     # cmd and alternative cmds are stored in list of list instead of dic to
     # maintain proper order for output
 
-    sys_shell_cmds = [
+    sys_shell_cmds: list[list[str]] = [
         ["hostname -I", "hostname"],
         ["top -n3 -b", "top -l 3"],
```

Comment:
differs with the command used in node.py, not sure if that is ok.

Reply:
I explained in my first comment why they are different. Yeah, merging them could be a possibility one day but that day is not today